### PR TITLE
feature: add make_onnx_model.py

### PR DESCRIPTION
**Pull request by YuriCat**
#233 enables us to use ONNX model created by this script.

### CODE REVIEW

File: scripts/make_onnx_model.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    args = yaml.safe_load(f)
+
+env = make_env(args['env_args'])
+model = env.net()
+model.load_state_dict(torch.load(model_path), strict=False)
+model.eval()
+model = ModelWrapper(model)
+
+model.load_state_dict(torch.load(model_path), strict=False)
+model.eval()
```
Comment (ikki407) [inline scripts/make_onnx_model.py:28]: Is model loading needed twice?

Comment (YuriCat) [reply scripts/make_onnx_model.py]: Thanks! removed

**Comment by YuriCat** [comment]
Since turn() is not always defined, turns()[0] is better.
I wonder there is any cases that no player acts in the original state.

File: scripts/make_onnx_model.py (hunk 1 above)
Comment (YuriCat) [reply scripts/make_onnx_model.py]: stopped importing and using ModelWrapper since it is used only for init_hidden()

**Comment by ikki407** [comment]
At initial state, a chance node perhaps exist depending on the implementation of the game (e.g. coin toss).

**Pull request merged**